server/src/controllers/temp-outfitter.es6

```diff
@@ -666,38 +666,39 @@ tempOutfitter.create = (req, res) => {
  * @param {Object} req - http request
  * @param {Object} res - http response
  */
-tempOutfitter.getOne = (req, res) => {
-  TempOutfitterApplication.findOne({
-    where: {
-      app_control_number: req.params.id
-    }
-  })
-    .then(app => {
-      if (!app) {
-        return res.status(404).send();
+tempOutfitter.getOne = async (req, res) => {
+  try {
```

Comment:
my only concern here unless its different for async/await is that if you mix a try/catch with a promise you can get an unhandled promise rejection warning.

Reply:
I think i now understand this enough to say we can start implementing where it makes sense.